.github/actions/update-activiti-cloud-application-base-tag/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Update Activiti Cloud Application base tag
+description: Update Activiti Cloud Application base tag in the release descriptor file
+inputs:
+  tag:
+    description: the value to be used as base tag for Activiti Cloud Application
+    required: true
+runs:
+  using: composite
+  steps:
+    - name: Update base tag
+      shell: bash
+      env:
+        ACTIVITI_CLOUD_APPLICATION_BASE_TAG: ${{ inputs.tag }}
+      run:
+        yq -i e '.release.baseTag.activitiCloudApplication = env(ACTIVITI_CLOUD_APPLICATION_BASE_TAG)' release.yaml
```

Comment:
To be honest, I asked myself exactly the same question, but I kept them separated only because initially there was a case where I wanted to update only one of them and I didn't want to add plenty of `if`s in the code. In the end, here, I always set all three values at the same time. However, if I move those to build-tools, I can reuse them in [activiti-cloud-application](https://github.com/Activiti/activiti-cloud-application/pull/744/files#r972939326), and in this case, I should be able to update only one of them.
Note that the tag name is already an input parameter, only the path to the property to be updated is not. I didn't manage to get `yq` working passing the path as variable, but it's maybe possible with some magics in bash.

Reply:
Ok understood, disregard if you have issues implementing it using a variable

Reply:
I found a way using `eval` and I created a generic action in build tools. See https://github.com/Alfresco/alfresco-build-tools/pull/130.